### ISSUE TITLE
duckduck go rate limite 202

### ISSUE THREAD

**Issue by buts101** [mamei16/LLM_Web_search_OWUI#1]
still persists after upgrade to 6.3.5

**Comment by mamei16** [comment]
Yeah I'm getting this error too sporadically, but there's not much than can be done, except using SearXNG instead. See the last reply in this issue thread: https://github.com/deedy5/duckduckgo_search/issues/254
